x-pack/legacy/plugins/canvas/i18n/components.ts

```diff
@@ -12,5 +12,110 @@ export const ComponentStrings = {
       }),
+    getTitleText: () =>
+      i18n.translate('xpack.canvas.embedObject.titleText', {
+        defaultMessage: 'Embed Object',
+      }),
+  },
+  Asset: {
+    getCopyAssetTooltipText: () =>
+      i18n.translate('xpack.canvas.asset.copyAssetTooltipText', {
+        defaultMessage: 'Copy id to clipboard',
+      }),
+    getCreateImageTooltipText: () =>
+      i18n.translate('xpack.canvas.asset.createImageTooltipText', {
+        defaultMessage: 'Create image element',
+      }),
+    getDeleteAssetTooltipText: () =>
+      i18n.translate('xpack.canvas.asset.deleteAssetTooltipText', {
+        defaultMessage: 'Delete',
+      }),
+    getDownloadAssetTooltipText: () =>
+      i18n.translate('xpack.canvas.asset.downloadAssetTooltipText', {
+        defaultMessage: 'Download',
+      }),
+    getThumbnailAltText: () =>
+      i18n.translate('xpack.canvas.asset.deleteAssetTooltipText', {
+        defaultMessage: 'Asset thumbnail',
+      }),
+  },
+  AssetManager: {
+    getBtnText: () =>
+      i18n.translate('xpack.canvas.assetManager.buttonText', {
+        defaultMessage: 'Manage assets',
+      }),
+    getConfirmModalBtnText: () =>
+      i18n.translate('xpack.canvas.assetManager.confirmModalButtonText', {
+        defaultMessage: 'Remove',
+      }),
+    getConfirmModalMessageText: () =>
+      i18n.translate('xpack.canvas.assetManager.confirmModalMessage', {
+        defaultMessage: 'Are you sure you want to remove this asset?',
+      }),
+    getConfirmModalTitleText: () =>
+      i18n.translate('xpack.canvas.assetManager.confirmModalTitleText', {
+        defaultMessage: 'Remove Asset',
+      }),
+  },
+  AssetModal: {
+    getDescriptionText: () =>
+      i18n.translate('xpack.canvas.assetModal.descriptionText', {
+        defaultMessage:
+          'Below are the image assets in this workpad. Any assets that are currently in use cannot be determined at this time. To reclaim space, delete assets.',
+      }),
+    getEmptyAssetsMessageText: () =>
+      i18n.translate('xpack.canvas.assetModal.emptyAssetsMessage', {
+        defaultMessage: 'Import your assets to get started',
+      }),
+    getFilePickerPromptText: () =>
+      i18n.translate('xpack.canvas.assetModal.filePickerPromptText', {
+        defaultMessage: 'Select or drag and drop images',
+      }),
+    getLoadingText: () =>
+      i18n.translate('xpack.canvas.assetModal.loadingText', {
+        defaultMessage: 'Uploading images',
+      }),
+    getModalCloseBtnText: () =>
+      i18n.translate('xpack.canvas.assetModal.modalCloseButtonText', {
+        defaultMessage: 'Close',
+      }),
+    getModalTitleText: () =>
+      i18n.translate('xpack.canvas.assetModal.modalTitleText', {
+        defaultMessage: 'Manage workpad assets',
+      }),
+    getSpaceUsedText: (percentageUsed: number) =>
+      i18n.translate('xpack.canvas.assetModal.spacedUsedText', {
+        defaultMessage: `${percentageUsed}% space used`,
```

Comment:
This should have a value, not use template strings:

```
defaultMessage: '{percentageUsed} space used',
values: {
  percentageUsed
}
```

https://github.com/elastic/kibana/blob/master/src/dev/i18n/README.md